workflows/phaseimpute/main.nf

```diff
@@ -262,8 +243,11 @@ workflow PHASEIMPUTE {
             if (params.tools.split(',').contains("stitch")) {
                 print("Impute with STITCH")
 
+                // Use provided posfile
                 if (params.posfile) {
-                    ch_posfile_stitch = ch_posfile
+                    //ch_posfile_stitch = ch_posfile.stitch
+                    ch_posfile_stitch = POSFILE_PREPARE_GAWK(ch_posfile)
+                    POSFILE_PREPARE_GAWK.out.posfile.dump(tag:"POSFILE_PREPARE_GAWK")
```

Comment:
I've never used this operator.
What it's use ?

Reply:
Removed! It is used when running the pipeline with `-dump-channels`